pkg/sql/logictest/testdata/logic_test/alter_column_type

```diff
@@ -30,6 +30,45 @@ statement ok
 DROP TABLE t
 
 
+# Demonstrate manual flow for non-trivial column change
+subtest ManualGeneralChange
+
+statement ok
+CREATE TABLE t (a INT PRIMARY KEY, b STRING)
+
+statement ok
+CREATE INDEX idx ON t (b)
+
+statement ok
+INSERT INTO t VALUES (1, '10'), (2, '20'), (3, '30')
+
+statement ok
+ALTER TABLE t ADD COLUMN i INT as (b::INT + 5) STORED
+
+statement ok
+CREATE INDEX idx2 ON t (i)
+
+statement ok
+ALTER TABLE t DROP COLUMN b CASCADE, ALTER COLUMN i DROP STORED
```

Comment:
Wow I'm surprised you can drop a column that a computed column depends on. What is the output of SHOW CREAT TABLE after this statement?